fairgame.js

```diff
@@ -390,8 +390,29 @@ function setLadderRows() {
     clientData.ladderPadding = qolOptions.expandedLadder.size / 2;
 }
 
+
+function expandLadder() {
+    if(document.getElementsByClassName("ladder-container").length > 0) {
+        return;
+    }
+    var input = 10000;
```

Comment:
This could lead to some client lag, better keep it at 250-500 or so?

Reply:
250 -500 would not give us the entire ladder.
It should only lag if we have actually that many rankers. I will add an option for the input size.